website/docs/src/content/quickstart.mdx

```diff
@@ -341,10 +341,233 @@ const stream = await workflow.run({ topic: "latest quantum computing chips" });
 
 ![blog writing trace](/quickstart/blog-trace.png)
 
-## Next steps
+## Running the dev server
```

Comment:
Not worth doing now, but at some point we should probably convert this to a multipage quickstart. It's very overwhelming in it's current form. Doesn't feel so "quick" with all the new additions.

Reply:
Agree we should do this. At a certain point we might want to just focus the quickstart on doing an immediate deployment. 

Opened https://github.com/gensx-inc/gensx/issues/597 to track a future update.